src/solver.cpp

```diff
@@ -580,11 +582,13 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     Register(registry, ++id, Primitive::Fusion, fusion::BnFwdInferActivationFused{}.SolverDbId());
     Register(registry, ++id, Primitive::Fusion, fusion::BnFwdTrgActivationFused{}.SolverDbId());
     Register(registry, ++id, Primitive::Fusion, fusion::BnBwdTrgActivationFused{}.SolverDbId());
+#if MIOPEN_USE_COMPOSABLEKERNEL
     Register(registry,
              ++id,
              Primitive::Fusion,
              fusion::ConvCKIgemmFwdBiasActivFused{}.SolverDbId(),
              miopenConvolutionAlgoImplicitGEMM);
+#endif
```

Comment:
This definitely breaks stable monotonic `id` which must be compatible across all of the MIOpen versions.